### ISSUE TITLE
[PROPOSAL] The New Chapter In The Saber Saga On Solana

### ISSUE THREAD

**Issue by c2yptic** [saber-hq/governance#21]
Greetings, everyone!

It's C2yptic here, and I'm excited to share with you my thoughts and ideas for future developments in the Saber saga. As some of you may know, I’m a total Solana maxi, and I've been a part of the Solana community since early 2021 and have been involved in projects like Marinade.finance and joinPoolParty.io, as well as ChronoKingdoms.com. 

Now, I've taken it upon myself the idea of assisting the Saber community and SBR token holders in moving things forward with making Saber a fully community-driven DAO protocol and making the protocol reach its ambitions of a “curve model” on Solana. 

I think Saber is an excellent DeFi protocol on Solana, that with the right leadership can take upon itself any challenges. Over the past 6-12 months, we've seen that the protocol has operated smoothly without developer involvement. However, it saddens me to see that there has been a lack of communication and leadership to make Saber a truly community-driven protocol under the DAO umbrella and to reach its goal of being a DAO-governed curve of Solana.

To put my view of the Saber protocol in simpler terms:

_Saber is like a beautiful race car that has been sitting in a parking lot, collecting dust for far too long. Although its appearance may not be appealing at the moment, the car still runs smoothly like a race car should. My aim is to clean it up and make it look shiny and attractive again, with the hope that those who were initially enthusiastic about this new type of car will appreciate it once more._

I reached out to and talked with Ian, who said that he will continue to contribute to the DAO and its protocol with any technical knowledge and aspects. I’ve also reached out to several DAO members, AKA SBR token holders to hear their points of view on my ideas and initiative, and so far I’ve only heard positive things related to my initiative. 

I feel confident that I have the experience, competence, and means to be able to revive the Saber DAO with my leadership and make the SBR token holders as whole as possible and propel the Saber protocol forward to be fully run and governed by the DAO. 

**My first order of tasks** would be to make sure we **immediately** move forward with executing pending governance proposals and move a specific existing proposal to on-chain voting, which has to do with the LP fees. See the following:

- Issue 1 SBR each for sunSBR 
https://tribeca.so/gov/sbr/proposals/43 
- Move this proposal to on-chain voting (LP fees) 
https://github.com/saber-hq/governance/issues/19 

**Let’s look at the details of what I need...**

My proposal is as follows, I ask all the SBR token holders to grant me the mandate to: 

Take leadership of the Saber DAO, which means the following...

- Leading Saber DAO public communications ( https://twitter.com/The_Saber_DAO )
- Community management (Discord [http://discord.gg/KTMrMErE2K](https://t.co/ZUXxaI8CCd))
- Manage the governance forum (https://github.com/saber-hq/governance/issues ) 
- I will be added to the Multisig that governs the DAO and the protocol (as a security measure, please see “Executive Council”, and “Emergency Multisig”, in the [Tribeca docs](https://docs.tribeca.so/more-resources/recommended-configuration/))
- Create a community council of select SBR token holders, that can help with guidance and perspectives until a DAO executive board has been formed (see below).
- Be able to, with executive authority, see through any needed changes. That means giving me the mandate of somewhat like a Chairman or President of the DAO. 
- Create the first DAO board AKA DAO team. Note that later on the DAO will be having all executive members appointed by voting, including my leadership role when the restructuring of the community and governance is completed and tested. A great example of this would be MonkeDAO’s governance structure. I will not be able to do everything by myself, so I will reach out to the Solana community, and to those with the relevant experiences to help out in making Saber reach its true DAO ambitions by forming the first DAO Executive board. 

+ I ask to be entrusted with this leadership role for an 8-month period, to begin with, to be able to see through the restructuring, organizing, and relevant efforts without any stress or unnecessary pressure. 

I do not ask for any SBR token compensation at all to begin with. Instead, I will acquire more SBR tokens on the open markets myself, to show the DAO my commitment to the success of the DAO and the protocol.  I will revisit this after I’ve accomplished the expectations set in this proposal. I envision DAO members, through a governance proposal will vote and decide what would be a fair and relevant SBR token compensation for past and possibly more work. I think this is the most fair approach for the DAO and me. 

I will do my utmost to make sure that this process of opening up the new chapter in the Saber saga is as smooth and fully transparent. Big decisions will not be taken out of the blue, but certain actions will have to be made with certain executive authority granted to me by the DAO through this proposal in order to be able to shape things up effectively. I will make monthly updates on all progress. 

I genuinely believe that these changes that I’ve outlined will be beneficial for Solana, the SBR token holders, and the Saber community as a whole.

SBR token holders, let's move into the second half of 2023 with new energy and meet the ultimate goal of making Saber fully run and governed by the DAO and its SBR token holders. Together we can create a new and glorious chapter in the Saber saga which will benefit not only SBR token holders, the Saber protocol… but also the Solana ecosystem as a whole!

**Comment by acamill** [comment]
Hi, we are glad to see some initiative on the Saber front! Thanks for taking the time to write this.

Leadership is fine, but unilateral executive power doesn't seem very adequate (although it depends on what is exactly implied by executive power).
We are aware most projects use keys directly held by members but we should strive to use multi-sigs, which benefits the ecosystem. Here we see a direct low "hanging fruit" improvement by doing so.

At UXD we have, since day one, held all our assets in our solana-governance DAO (multisig, council/gov token mix), and have been pushing constantly for more protocols integration on Realm. 
As such, we understand from experience that it is tedious to have ALL proposals voted through gov token, there is a lot of "run" day to day operations that are of no interest for voters, and also voters are not always voting.
The compromise we have in place is that the Program Upgrade Authority is held under UXP token holder control, but other "satellite" governances handling mundane operations and fund management are held under the council token authority held by the team members. 
Note that for important protocol-wise decisions, we hold opinion votes using UXP governance.

Would it be possible to do something similar here with Saber? And if missing reliable members to be part of this council, we propose our participation (or the participation of any other long standing, incentives aligned, project of the Solana ecosystem).
For instance, that's what we are doing for controlling the upgrade authority of the Solana-governance program public instance (that we use for our DAO).


On top of that, UXD protocol, through its DAO, holds a considerable chunk of voting power over Saber, as it implemented "automatic" (still voting and creating proposals weekly on our DAO) claim/locks/gauge votes.
As such we would like to possibly extend a vested grant for your efforts, as it is in the best interest of everyone in the ecosystem, and for UXD Protocol, to keep Saber sturdy and up to date.
Rest assured that our incentives are fully aligned with Saber as we have all our tokens 4 years locked.

We have a few questions about the execution of your plan. 
What is your vision for Saber and what kind of "needed changes" do you foresee short term for the protocol (beyond the fee increase and the sunSBR conversion, both of which we support). 
We are more interested about the "what" than the "how" at this stage if that makes sense. 

And... Finally before all this, I think it's important to look at the context:
- Saber program upgrade authority, held by the Saber DAO on Tribeca
- Saber DAO on Tribeca controlled by Saber token vote (and ve)
- Tribeca program upgrade authority, held by a multisig controlled by Saber/Marinade team(?)
- Saber DAO treasury is held by Ian on a ledger
- SBR token mint (held by the mint-wrapper) controlled by the Saber DAO

We believe the first step would be to :
- add 3 solana ecosystem serious stakeholders to the Tribeca program upgrade authority, that way it becomes a 3:5
- repatriate the DAO treasury under Saber DAO, or the aforementioned multisig in the interim if the Tribeca program doesn't support tokens

Once this is established, we think that you spearheading this effort is a very good thing, and should be rewarded with a Saber DAO grant and/or grant from UXD Protocol if we can promote the development of Saber Protocol (in SBR or UXP).
Although two points:
- would need to know what are the plans (apart from the two existing proposal that we support)
- would need to be through the existing framework of the DAO (no need for special executive power/role, the role is taken through contribution by anyone DAO style)

Cheers, UXD Team

**Comment by c2yptic** [comment]
Thanks for participating in this important discussion! 

> - add 3 solana ecosystem serious stakeholders to the Tribeca program upgrade authority, that way it becomes a 3:5
> - repatriate the DAO treasury under Saber DAO, or the aforementioned multisig in the interim if the Tribeca program doesn't support tokens

I agree that this is very much needed. I reached out to the Marinade team who are collaborative and willing to help out with the necessary steps related to Tribeca since they are [migrating their governance from Tribeca to Realms ](https://forum.marinade.finance/t/mdao-proposal-final-phase-of-the-migration-to-realms/523)

> - would need to know what are the plans (apart from the two existing proposal that we support)
> - would need to be through the existing framework of the DAO (no need for special executive power/role, the role is taken through contribution by anyone DAO style)

With the mandate requested in my proposal, I would be able to take the measures required to move SaberDAO into a proper DAO in a quick and efficient manner. I think that MonkeDAO's current governance structure is a great way to go about it (with an executive board that has been voted into power) and this is my vision and ambition for SaberDAO. The plan is to first and foremost make a robust governance structure with DAO contributors receiving the mandate from the DAO to be part of an executive board as outlined here "Create the first DAO board AKA DAO team." in the proposal. 

To me, it makes sense that parties such as the UXD team, who have a substantial amount of SBR tokens and with that a vested interest in the long-term success of the SaberDAO and its protocol, to participate in this first edition of the executive board until we have a rigid process to elect the executive board members by vote. I look forward to being able to tailor these details with parties such as the UXD team!

**Comment by c2yptic** [comment]
With the overwhelming support received from the community related to my initiative, I moved the proposal on-chain:

https://tribeca.so/gov/sbr/proposals/47

**Comment by acamill** [comment]
This doesn't address the topics raised in the previous post. 

- What is the exact scope of the "mandate"? Would you be holding any keys? Who would discuss the choices?
- What are these actual "take the measures required to move SaberDAO into a proper DAO"?
- Why the current DAO infrastructure in place isn't sufficient? 
- Why the urgency

I think first and foremost the shortcomings highlighted above (multisig 3:5 and treasury) should be addressed, it can be done very quickly. 
Then we should resume this discussion. The proposal seems very precipitated to me, why the urgency.

It's all very loosely defined and what this proposal basically means is: 
- transfer Saber direction to @c2yptic
- let @c2yptic decide of the direction to take (**unknown yet**)
- let @c2yptic setup the DAO analogous to MonkeDAO (**what are the current shortcomings? What would that bring**)

I'm all for moving to Realm instead of Tribeca, which I'm very familiar with. But I don't think this is the scope of this discussion and maybe we should focus on the core.

Also, any tweet or reference to that overwhelming support you could share

**Comment by aperire** [comment]
Would certainly vote for it

**Comment by c2yptic** [comment]
**The vote passed! Now we'll have a new chapter in the Saber saga!**

https://tribeca.so/gov/sbr/proposals/47 

I am closing this thread, but I will happily answer any questions or discuss further ideas in the SaberDAO discord: https://discord.com/invite/29teHrsYJD
